src/coreclr/jit/lsrabuild.cpp

```diff
@@ -1899,6 +1899,18 @@ void LinearScan::buildPhysRegRecords()
         RegRecord* curr = &physRegs[reg];
         curr->regOrder  = (unsigned char)i;
     }
+#if defined(TARGET_AMD64)
+    if (compiler->DoJitStressEvexEncoding())
```

Comment:
EVEX is enabled by default now, so this should be:
```suggestion
    if (compiler->canUseEvexEncoding())
```

Reply:
comment resolved.